build.bat

```diff
@@ -53,6 +53,7 @@ MKDIR "%NUGET_FRAMEWORK_FOLDER%" ||  GOTO BuildFailed
 XCOPY /F "%PROJECT_FOLDER%\bin\Release\RemoteTea.Net.*" "%NUGET_FRAMEWORK_FOLDER%\" ||  GOTO BuildFailed
 XCOPY /F "%PROJECT_FOLDER%\bin\Release\RemoteTea.Net.*" "%NUGET_PACKAGE_FOLDER%\Tools\" ||  GOTO BuildFailed
 XCOPY /F "%PROJECT_FOLDER%\bin\Release\%NUGET_PACKAGE_ID%.nuspec" "%NUGET_PACKAGE_FOLDER%\" ||  GOTO BuildFailed
+echo f | XCOPY /F /Y "%PROJECT_FOLDER%\..\LICENSE.txt" "%NUGET_PACKAGE_FOLDER%\license\LICENSE.txt" ||  GOTO BuildFailed
```

Comment:
Build error: unable to find `LICENSE.txt`.
Another option to consider is to use an expression instead of a file:
`<license type="expression">GPL-2.0-only</license>`